watchers/configmap_watcher.go

```diff
@@ -0,0 +1,60 @@
+package watcher
+
+import (
+	"context"
+	"os"
+	"sync"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var logger = log.FromContext(context.TODO())
```

Comment:
WDYT of replacing this logger with adding passing the logger (setupLog) from the main to this function ?

Reply:
The `setupLog` has name "setup"  ([main@108](https://github.com/jakub-dzon/k4e-operator/blob/338ec3af0d0ecc40626d90fae00c65b3d77bf487/main.go#L108)).
Is this ok for logger in `configmap_watcher.go`?

Reply:
one might claim that setting a watcher for the configmap is part of the operator's setup.